### PR TITLE
Refactoring loom-embed

### PR DESCRIPTION
**Pull request by sidraman14**
Refactoring loom-embed to remove embed code and keep https://loom.dev/ running. 

https://www.loom.com/share/e96a74b95ad74700887ed5014e0e9449

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@loomhq/loom-embed",
+  "name": "@loomhq/loom-embed-old",
```
Comment (brendonjohn) [inline package.json:2]:
```suggestion
  "name": "@loomhq/loom-embed-docs",
```

File: src/sdk/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import linkReplace from './linkReplace';
```
Comment (brendonjohn) [inline src/sdk/index.js:1]: I think we should remove this file too.

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
   "version": "1.1.7",
   "description": "SDK for easy embedding videos",
   "main": "dist/sdk.js",
```
Comment (brendonjohn) [inline package.json:5]:
```suggestion
```

I don't believe we require `main` to be defined.

**Review by brendonjohn — COMMENTED** [review]
Nice progress. Thanks Sid.

I think we should replace loom-embed with `@loomhq/loom-embed` here too.
https://github.com/loomhq/loom-embed/blob/6774f14cf2fcdb007bcf66c5ae26651b5bbc1dc6/src/examples/components/OembedExample.js#L3

There might be more instances of where `loom-embed` needs to be replaced.

**Review by brendonjohn — APPROVED** [review]
lgtm;

Let's get this merged.

Do you have a plan for publishing the examples/docs?

**Comment by sidraman14** [comment]
> Do you have a plan for publishing the examples/docs?

Not currently, I was planning on working more on the SDK, and was told there would be an overhaul of documentation later on. However, if you think it worth working on now I would be willing to work on that as well.

**Pull request merged**